spec/spec_helper.rb

```diff
@@ -1,19 +1,3 @@
-if ENV['COVERAGE'] == 'true'
```

Comment:
We could consider keeping `simplecov` since it protects against missing coverage that can be introduced by running only differential mutations. If you still prefer to remove it though I don't think it's especially critical.